scripts/main.coffee

```diff
@@ -13,13 +13,14 @@ module.exports = (robot) ->
 
   config = require('../config.json')
   messages = require('./messages.js').messages
+  formatting = require('./formatting.js').formatting
 
   # messages.setAuth((err, res) ->
   #     console.log("Logged in: " + res[1].title)
   # )
 
   robot.hear /.*/i, (res) ->
-      message = messages.getAction(res.message.text)
+      message = formatting.getDataMask(res.message.text, /\+[^*\s]+/)
```

Comment:
Is this line related to tests? why have you changed the name, found the bug? if so I suggest doing this in separate commit, cause it is really confusing.